contents/java/JavaReflections.md

```diff
@@ -1,173 +1,160 @@
 
 Take for example the following class:
 
 ```java
 public class Student {
 
-	private final String name;
-	private final String gender;
+  private final String name;
+  private final String gender;
 
-	public Student(String name, String gender) {
-		this.name = name;
-		this.gender = gender;
-	}
+  public Student(String name, String gender) {
+    this.name = name;
+    this.gender = gender;
+  }
 
-    // Other methods here...
+  // Other methods here...
 }
 ```
 
-An instance of a class can be created as usual using the `new` keyword
+An instance of `Student` class can be created as usual using the `new` keyword:
 
 ```java
 Student john = new Student("John Doe", "Male");
 ```
 
-However, it is also possible to get information about the class itself because the class itself is an `Object` like so:
+However, it is also possible to get information about the `Student` class itself because the class itself is an `Object`:
 
 ```java
 Class<Student> studentClass = Student.class;
 ```
 
-It also means that you can store any `Class` object in any data structure for future retrieval. This is the main entry
-point for Java's reflections. You can now get the name of the class, create new instances of the class, observe its
-public/private fields - the possibilities are endless!
+This means that you can store any `Class` object in any data structure for future retrieval. This is the main entry point for Java's reflections. You can now get the name of the class, create new instances of the class, observe its public/private fields - the possibilities are endless!
 
 ### Getting Started
 
-There are many webpages dedicated to explaining the details of reflections in Java; so this will not repeat what is
-being made readily available on the web. One good place to start is [http://www.javaworld.com/article/2077015/java-se/take-an-in-depth-look-at-the-java-reflection-api.html](http://www.javaworld.com/article/2077015/java-se/take-an-in-depth-look-at-the-java-reflection-api.html)
+There are many webpages dedicated to explaining the details of reflections in Java; so this will not repeat what is being made readily available on the web. One good place to start is this [article by JavaWorld](http://www.javaworld.com/article/2077015/java-se/take-an-in-depth-look-at-the-java-reflection-api.html).
 
-One important point is that while Java reflections are powerful, its implementations are not very straightforward.
-There are however some libraries out there to make life easier such as the [Google's Guava
-library](https://github.com/google/guava/wiki/ReflectionExplained) which contains many utility methods and classes to
-make your life easier.
+One important point is that while Java reflections are powerful, its implementations are not very straightforward. There are however some libraries out there to make life easier such as the [Google's Guava
+library](https://github.com/google/guava/wiki/ReflectionExplained) which contains many utility methods and classes.
```

Comment:
Extra line break?

There are however some libraries out ~~there to make life easier~~ such as the [Google's Guava
library](https://github.com/google/guava/wiki/ReflectionExplained) which contains many utility methods and classes **that can make our life easier**.